homework/content.js

```diff
@@ -1,37 +1,45 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+  
+  var ToDoContentManager = function () {
+      this._wrapper = null;  
+  };
+  ToDoContentManager.prototype = {
+    start(){
+        /*
+         *Initiate wrapper.
+         *To do bind(this)
+         */
+        this._wrapper = document.querySelector('note-content-wrapper');
+        window.addEventListener('note-open',(function(event){
```

Comment:
Rather than binding on an anonymous function, you need to bind the `this` via set it as the listener of `addEventListener`.